portfolio/src/main/webapp/script.js

```diff
@@ -50,3 +50,13 @@ function changeCat() {
   const catImage = document.getElementById('cat-image');
   catImage.src = src;
 }
+
+function hello() {
```

Comment:
Is there a better name that we could use, that would be more descriptive for this function.  This may make it a little easier to read, to understand what it is doing.